cmake/modules/RootBuildOptions.cmake

```diff
@@ -301,6 +305,18 @@ endif()
 #---Define at moment the options with the selected default values-----------------------------
 ROOT_APPLY_OPTIONS()
 
+if(root7)
+  if(NOT CMAKE_CXX_STANDARD)
+      set(CMAKE_CXX_STANDARD 14 CACHE STRING "C++14 standard used with root7")
+      message(STATUS "Enable C++14 for compilation of root7 components")
```

Comment:
"Enabling" is clearer; "enable" could be a directive to the user ("re-run cmake, but then please enable...")